.travis.yml

```diff
@@ -81,31 +69,10 @@ matrix:
     - rvm: jruby-head
       gemfile: gemfiles/postgresql/4-2.gemfile
     - rvm: jruby-head
-      gemfile: gemfiles/postgresql/5-0.gemfile
+      gemfile: gemfiles/postgresql/5-1.gemfile
     - rvm: jruby-head
       gemfile: gemfiles/sqlite3/4-2.gemfile
     - rvm: jruby-head
-      gemfile: gemfiles/sqlite3/5-0.gemfile
+      gemfile: gemfiles/sqlite3/5-1.gemfile
 
-    # legacy testing
-    # things still run and we don't have a good reason to break it
-    # but we don't need to keep running the full matrix
-    - gemfile: gemfiles/mysql/3-0.gemfile
-      rvm: 2.0.0
-    - gemfile: gemfiles/mysql/3-1.gemfile
-      rvm: 2.0.0
-    - gemfile: gemfiles/mysql/3-2.gemfile
-      rvm: 2.0.0
-    - gemfile: gemfiles/postgresql/3-0.gemfile
-      rvm: 2.0.0
-    - gemfile: gemfiles/postgresql/3-1.gemfile
-      rvm: 2.0.0
-    - gemfile: gemfiles/postgresql/3-2.gemfile
-      rvm: 2.0.0
-    - gemfile: gemfiles/sqlite3/3-0.gemfile
-      rvm: 2.0.0
-    - gemfile: gemfiles/sqlite3/3-1.gemfile
-      rvm: 2.0.0
-    - gemfile: gemfiles/sqlite3/3-2.gemfile
-      rvm: 2.0.0
```

Comment:
@laserlemon I was specifically keeping around the legacy section for exactly the reason the comment states. I would prefer to continue to do so.